### DISCUSSION TITLE
HyperHDR with USB/COM and Low fps

### DISCUSSION THREAD

**Discussion by nebka94380** [awawa-dev/HyperHDR#413] [Q&A]
hello I just installed HyperHDR on my pc with a usb cable and com port but I see that I do not have many frames or fps. I have latencies of less than a second. can i add more speed?

Thank you sorry I'm brand new.

Baudrate : 115200
https://www.zupimages.net/viewer.php?id=22/45/5ja1.jpg
https://www.zupimages.net/viewer.php?id=22/45/l3v5.jpg

**Comment by awawa-dev** [comment]
Hi
115200 is a very, very low speed. Modern solutions offer x20 and even x40 faster speed. Number of LED and their type (sk6812 RGBW needs more resources) affect LED refresh rate. Please read more here https://github.com/awawa-dev/HyperSerialEsp8266 (also there is a version for ESP32 https://github.com/awawa-dev/HyperSerialESP32). They require ESP boards and a level shifter 3.3v to 5v is recommended, Arduino should handle ~500000 speed and should fit for small setups (it already operates at 5v).

**Comment by nebka94380** [comment]
Thank you for your answer.
so I bought the small LED BOX on Ali¤¤¤¤ for nothing.

I can't find a tutorial on how to make one.

which plate should I take for a 32" tv

Thanks

**Comment by awawa-dev** [comment]
I guess it's a arduino inside. Maybe you can find a flash using 500000 speed. Or you buy Arduino with 340G and use a sketch for Adalight. 32"TV is a small small but as I said: everything depends on LED number and their type. Arduino setups are usually simpler and you can find some tutorials on the net, I'm focusing on ESP boards.

**Comment by nebka94380** [comment]
I just bought an ESP8266 everything is nikel but I'm still at 10 Fps while in Baudrate 230400 is this normal?

**Reply by awawa-dev** [reply]
How do you know you exactly have around 10fps? If you don't want to share essentional information about you setup then I can't help.

**Comment by nebka94380** [comment]
In performance 

https://zupimages.net/up/22/45/nyln.jpg

**Reply by awawa-dev** [reply]
This is performance of your video source, not LED driver like esp8266 (which is absent btw)
